.github/workflows/ubuntu-openblas.yml

```diff
@@ -32,123 +32,26 @@ jobs:
           source util/ci_utils.sh
           maximize_ubuntu_github_actions_build_space
       - name: Docker build
-        run: docker/docker_build.sh openblas-amd64-py310-dev
+        run: docker/docker_build.sh openblas-amd64-py312-dev
```

Comment:
Can we make this a workflow input (and keep 312 as default). That will make it easy for people to create wheels for their python versions. See the `ubuntu-wheel` workflow as an example. An additional input would be DEVELOPER_BUILD. Also see that workflow to upload the generated wheel to github as an artifact. This makes ARM64 developer wheels available.

Reply:
done